fava/template_filters.py

```diff
@@ -171,3 +171,20 @@ def format_errormsg(message):
     new_message = re.sub(ACCOUNT_RE,
                          '<a href="{}">'.format(url) + r'\1' + '</a>', message)
     return new_message.replace('for \'', 'for ').replace('\': ', ': ')
+
+
+def date_group(entry):
+    """Return the date group an entry belongs to."""
+    kind = g.ledger.fava_options['interval']
+    date = entry.date
+    if kind == 'year':
+        return date.year
+    elif kind == 'quarter':
+        return '{}-Q{}'.format(date.year, (date.month-1)//3+1)
+    elif kind == 'month':
+        return '{}-{:02}'.format(date.year, date.month)
```

Comment:
just return the tuple, same for the other cases

Reply:
https://github.com/beancount/fava/pull/620#discussion_r159632840 ?